### PR TITLE
#113396 Results of upgrade RIPS to Valtimo 12.0 RC

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: release-notes/major12/12.0.0/migration.md [hunk 1]
```diff
@@ -12,8 +12,11 @@ This page describes how to update Valtimo from the previous version to the curre
 
   Scope: back-end
 
-  `valtimo-dependencies` and `valtimo-gzac-depenencies` should no longer be used for `dependencyManagement` or `platform`.
+  1. `valtimo-dependencies` and `valtimo-gzac-depenencies` should no longer be used for `dependencyManagement` or `platform`.
```
Comment (Hildebrand-Ritense) [inline release-notes/major12/12.0.0/migration.md:15]: valtimo-gzac-depenencies - is that a typo or is that really the name?

File: release-notes/major12/12.0.0/migration.md [hunk 2]
```diff
@@ -219,9 +222,10 @@ This page describes how to update Valtimo from the previous version to the curre
         `ng update @angular/core@17 @angular/cli@17 --force` and commit all changes after the upgrade.
 
     3. **Step 3: @valtimo/form-link to @valtimo/process-link**
-       The `form-link` library has been renamed to `process-link`. Change `@valtimo/form-link` to `@valtimo/process-link`
-       in your `package.json` file. Also, replace all imports in your from `@valtimo/form-link` to
-       `@valtimo/process-link`.
+       The `form-link` library has been renamed to `process-link`. 
+       1. Change `@valtimo/form-link` to `@valtimo/process-link` in your `package.json` file. 
+       2. Replace all imports in your from `@valtimo/form-link` to `@valtimo/process-link`.
```
Comment (Hildebrand-Ritense) [inline release-notes/major12/12.0.0/migration.md:227]: in your what?

**Review by Hildebrand-Ritense — APPROVED** [review]
lgtm

File: release-notes/major12/12.0.0/migration.md [hunk 3]
```diff
@@ -12,8 +12,11 @@ This page describes how to update Valtimo from the previous version to the curre
 
   Scope: back-end
 
-  `valtimo-dependencies` and `valtimo-gzac-depenencies` should no longer be used for `dependencyManagement` or `platform`.
+  1. `valtimo-dependencies` and `valtimo-gzac-dependencies` should no longer be used for `dependencyManagement` or `platform`.
   This functionality has been replaced by [valtimo-dependency-versions](../../../getting-started/modules/core/valtimo-dependency-versions.md).
+  2. If present, remove the dependency to `org.hibernate:hibernate-envers` 
+  3. If present, upgrade dependency `org.hibernate:hibernate-core` to `6.0.2.Final` or higher.
+  4. If present, upgrade dependency `com.ritense.valtimo:valtimo-testcontainers` to `0.0.2-SNAPSHOT` or higher.
```
Comment (ThomasMinkeRitense) [inline release-notes/major12/12.0.0/migration.md:19]: The dependency isn't public so shouldn't be mentioned here imo

Comment (Klaas-Ritense) [reply release-notes/major12/12.0.0/migration.md:19]: I've thought about that. But having this in the migration notes, will prevent a lot of troubleshooting in our implementation teams .

Comment (ThomasMinkeRitense) [reply release-notes/major12/12.0.0/migration.md:19]: I see your point, but I think the downside of having this in the public docs is greater than the upside. Let's find out who uses this library and communicate directly to them.

Comment (Klaas-Ritense) [reply release-notes/major12/12.0.0/migration.md:19]: What are the downsides?

Comment (Klaas-Ritense) [reply release-notes/major12/12.0.0/migration.md]: Removed

**Pull request merged**